app/controllers/api/products_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::ProductsController < ApplicationController
+  def index
+    product = Product.all 
```

Comment:
please change the variable to plural, it is a more telling name since we are storing an array of several products in that variable

Reply:
variable is now plural